<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgColorToAlphaBase</class>
 <widget class="QWidget" name="WdgColorToAlphaBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>254</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="KoTriangleColorSelector" name="colorSelector" native="true"/>
       </item>
       <item>
        <widget class="KColorButton" name="btnCustomColor">
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel1_2">
     <property name="text">
      <string>Threshold:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="sldSecondaryThreshold">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="intThreshold">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KoTriangleColorSelector</class>
   <extends>QWidget</extends>
   <header>KoTriangleColorSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kcolorbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>sldSecondaryThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>intThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldSecondaryThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
